includes/utilities.inc

```diff
@@ -982,6 +982,13 @@ function islandora_paged_content_page_derivatives($context) {
       'function' => array('islandora_ocr_derive_hocr'),
       'file' => "$ocr_module_path/includes/derivatives.inc",
     );
+    $derivatives[] = array(
+      'source_dsid' => 'OBJ',
+      'destination_dsid' => NULL,
+      'weight' => -1,
```

Comment:
This appears to be weighted _before_ the other derivative implementations making use of the statements, which seems surprising at first... Should it be ordered after, or is there something going on behind the scenes (which would probably merit a comment in code)?

Reply:
@qadan @adam-vessey i second the question there: why before every other one? Lastly, if really that is the way maybe just move other derivatives one place up?

Reply:
That was definitely a mistake; I'll correct that